client/my-sites/backup/clone-flow/index.tsx

```diff
@@ -163,10 +167,17 @@ const BackupCloneFlow: FunctionComponent< Props > = ( { siteId } ) => {
 		if ( true === isNavigating ) {
 			const selectedSite = stagingSites.find( ( site ) => site.siteurl === newValue );
 			if ( selectedSite ) {
-				setCloneDestination( selectedSite.blog_id.toString() );
-				setUserHasSetDestination( true );
-				setIsCloneToStaging( true );
-				dispatch( recordTracksEvent( 'calypso_jetpack_clone_flow_set_staging_site' ) );
+				if ( selectedSite.role?.startsWith( 'staging-' ) ) {
+					setCloneDestination( selectedSite.role );
+					setUserHasSetDestination( true );
+					setIsCloneToStaging( false );
```

Comment:
I guess you are setting this to false, so we could use the regular `rewindClone()` action. Probably we might need to clean this clone flow component a bit on new iterations when we unify the clone to staging experience in the backend.

Reply:
Technically those are clone to staging too, but on the same site ID. So we could consider in the future something like `isCopyingSameSite` instead of `isCloneToStaging`.